docs/howto/override-project-templates.txt

```diff
@@ -0,0 +1,248 @@
+to adapt the settings reference in ``manage.py`` to refer to ``"settings"``
+instead of ``"myproject.settings"`` since your ``settings.py`` file is no
+longer in the created ``myproject`` folder, but at the same level as
+``manage.py``. Likewise in ``settings.py``, you'd have to change the value of
+:settings:`ROOT_URLCONF` from ``"myproject.urls"`` to simply ``"urls"`` and
+:settings:`WSGI_APPLICATION` from ``"myproject.wsgi.application"`` to
+``"wsgi.application"``.
+
+For the :djadmin:`startapp` command, there is less that can be customized since
+it is a single folder with files and a migrations folder within. If certain
+files are not needed they can be left out. For example, if no models will be
+used, the ``models.py`` file and the ``migrations`` folder can be left out. In
+this case the ``admin.py`` file could also be left off. Conversely, if you
+always use custom managers, you could add a ``managers.py`` file to your custom
+structure to make sure the file is always there.
+
+Template suffixes
+-----------------
+
+For convenience, python files used in the templates use the suffix ``.py-tpl``.
+During the process of creating the project or app files, the suffix is changed
+to ``.py``. However, this is not absolutely necessary, since any files with the
+``.py`` suffix can have the context changed by the templating engine if they
+contain template tags that are variables.
+
+
+Templating non-Python files
+---------------------------
+
+Since the commands that use the templates just copy files, you can have any
+type of file in the project or app template. This is especially useful if you
+want to set up frontend or infrastructure files within the project template.
+
+Any type of file is supported as long as it doesn't end with ``.pyo``,
+``.pyc``, or ``.py.class``. Any type of directory is supported as long as it
+doesn't begin with a period ``.`` (hidden directory) or is a ``__pycache__``
+file.
+
+You can even use the template engine to fill in ``{{ project_name }}`` in
+places where the project name would be commonly used by other apps, such as in
+Kubernetes or Docker files.
+
+
+Excluding parts of files from templating
+----------------------------------------
+
+The templating engine will overwrite any context variables found in the project
+or app template. Sometimes, for the purposes of documentation, you want to
+leave the variables visible in the produced app or project structure. To have
+``{{ django_version }}`` or ``{{ project_name }}`` still visible in the
+resulting file, you can wrap that part of the file, or even the whole file with
+the template tags ``{% verbatim %}`` and ``{% endverbatim %}``. The templating
+engine will end up removing them from the finished file.
+
+
+Using your templates
+====================
+
+The basic command to start your project using your new template is:
+
+.. console::
+
+    $ django-admin startproject myproject .
```

Comment:
I think this assumes your in that directory? Perhaps we should be using something like
```suggestion
    $ django-admin startproject --template=/path_to_my_templates/my_app_template myapp
```